airbyte-cdk/python/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.25.1
+Low-Code CDK: make DatetimeStreamSlicer.step as InterpolatedString
```

Comment:
As part of the new release process, the version is updated in the new GitHub workflow. You can check https://airbytehq-team.slack.com/archives/C02UF50V9HA/p1674655636577679?thread_ts=1674654710.391809&cid=C02UF50V9HA for more information. 

An example of the issue is this: you have [another PR](https://github.com/airbytehq/airbyte/pull/21956) at the same time that has the same release version. This will clash i.e. one of the release will fail. To avoid this, we should always release on master. This allows to have one source of truth for the version and not multiple (one per branch)

Reply:
fixed